Autoprefixer.py

```diff
@@ -48,8 +48,24 @@ def has_selection(self):
 				return True
 		return False
 
-	def get_setting(self, key):
-		settings = self.view.settings().get('Autoprefixer')
+	@staticmethod
+	def get_setting(view, key):
```

Comment:
Probably better to just make this a top-level function instead of a class method.